app/views/directives/notifications/heading.html

```diff
@@ -0,0 +1,24 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6">
+      <strong>{{notificationGroup.heading}}</strong>
+    </div>
+    <div class="col-md-6 text-right small">
+      <a ng-href="project/{{$ctrl.customScope.projectName}}/browse/events">All Events</a>
+    </div>
+  </div>
+  <div class="row mar-top-md">
+    <div class="col-md-6">
+      <em>{{$ctrl.customScope.countAllUnread(notificationGroup)}} Unread</em>
```

Comment:
Let's not count on every digest loop. Let's just count when the events change and set some scope var.